Assets/HoloToolkit/SpatialMapping/Scripts/RemoteMapping/Editor/RoomMeshExporter.cs

```diff
@@ -0,0 +1,246 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class RoomMeshExporter
+    {
+        private const string ExportDirectoryKey = "ExportDirectory";
```

Comment:
Yes, I figured this could be customized externally; hence the ExportDirectory property. Where would be a good location to put this field? Is there already a congregation of configurable HoloToolkit fields located somewhere?